tsconfig.eslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  // extend your base config so you don't have to redefine your compilerOptions
+  "extends": "./tsconfig.json",
+  "include": ["src/**/*.ts", "src/**/*.tsx", "test/**/*.ts", "./.eslintrc.js", "./webpack.config.js"]
```

Comment:
i'm not understanding the purpose of this file. why don't we want to include tests in the base tsconfig? and why DO we want to include estlintrc.js (which doesn't exist) or webpack.confg (which also doesn't exist)?

Reply:
whoops, that's a copy-paste mistake, the test folder also doesn't exist

Reply:
Just deleted this file, linter seems to be fine with it